tests_v2/test_background_workspace_idle_thread_footprint.py

```
@@ -0,0 +1,213 @@
+                f"budget={IDLE_FOOTPRINT_GROWTH_BUDGET_MB}MB",
+            )
+            _must(
+                client.current_workspace() == baseline_workspace,
+                "background workspace priming should not switch the selected workspace",
+            )
+        finally:
+            for workspace_id in reversed(created_workspaces):
+                try:
+                    client.close_workspace(workspace_id)
+                except Exception:
+                    pass
```

Ruff (0.15.12)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(Exception): ...`

(SIM105)